docs/env_vars_and_config.md

```diff
@@ -88,12 +169,72 @@ SetEnv ILIOS_MAILER_USER null
 SetEnv ILIOS_MAILER_PASSWORD null
 SetEnv ILIOS_LOCALE en
 
-SetEnvIf Host "ilios-staging\.example\.com" ILIOS_DATABASE_URL=SetEnv ILIOS_DATABASE_URL mysql://ilios_staging_db_user:Stag1ngPassw0rd@db-host1/ilios_db
-SetEnvIf Host "ilios-staging\.example\.com" ILIOS_DATABASE_USER=ilios_staging_db_user
+SetEnvIf Host "ilios-staging\.example\.com" ILIOS_DATABASE_URL=SetEnv ILIOS_DATABASE_URL mysql://ilios_staging_db_user:Stag1ngPassw0rd@db-host1/ilios_stage_db
 SetEnvIf Host "ilios-staging\.example\.com ILIOS_SECRET=ST@G1nGS3CRET12345
 
 SetEnvIf Host "ilios-production\.example\.com" ILIOS_DATABASE_URL=mysql://ilios_production_db_user:Pr0duct10nPassw0rd@/db-host1/ilios_production_db
 SetEnvIf Host "ilios-production\.example\.com ILIOS_SECRET=PR0DUCT10nS3CRET12345
 ```
 
 For more information on Apache Environments, `SetEnv`, and `SetEnvIf`, please refer to the Apache 2.4.x documentation at https://httpd.apache.org/docs/2.4/env.html
+
+## Verifying your Environment Variables
+
+#### Verifying the Environment at the Command Line
+
+If you would like to verify the env variables for yourself or a different user on command line, run the following commands:
+
+```bash
+# for yourself
+env
+```
+
+```bash
+# to check command line ENV vars for a different user:
+sudo -u [USER_NAME] env
+```
+
+#### Verifying the Environment on the Web Server
+To verify which ENV vars, if any, are set for within your web service, create a php file on your webserver with the following content and then view the file in your browser:
+
+```php
+<?php
+phpinfo();
```

Comment:
This can be seen at http://localhost:8000/ilios/health/ all top boxes will be green.